tests/nnfw_api/src/CircleGen.cc

```diff
@@ -205,6 +205,13 @@ uint32_t CircleGen::addOperatorSplit(const OperatorParams &params, int32_t num_s
                                 circle::BuiltinOptions_SplitOptions, options);
 }
 
+uint32_t CircleGen::addOperatorTile(const OperatorParams &params)
+{
+  auto options = circle::CreateRankOptions(_fbb).Union();
```

Comment:
`CreateRankOptions` -> `CreateTileOptions`

Reply:
Thanks! but it should be removed, as there is no `CreateTileOptions`.

Reply:
`circle::CreateTileOptions` :
`runtime/onert/frontend/circle_schema/include/circle_schema_generated.h:5528`

Reply:
Oops, I might looked at an old schema 😓